_posts/2025-05-02-consultas-rut.md

```
@@ -0,0 +1,21 @@
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [Reportes, Cliente, Ejecutivo, Deudor, Empresa, Matriz de riesgo]
+comments: false
+author: Catalina Pinilla
+---
+
+## Consulta por rut
+
+Desde ahora no es necesario que el rut que necesitas consultar este enrolado, porque ahora Advance cuanta con un apartado de `Consultas por rut` en donde podrás obtener los reportes de quien necesites consultar ya sea cliente de `Capital Express Servicios Financieros`, deudor o empresa. Puedes acceder a el desde la `Matriz de riesgo`:
+
+<p align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)